packages/arch/packages/hooks/src/index.ts

```diff
@@ -1,15 +1,17 @@
 import ResizeObserver from 'resize-observer-polyfill';
```

Comment:
I just changed this file to TS because it seemed like the smallest thing I could test things on.